duckduckgo/macos-browser
# Receiving data from Bitwarden fixed

Task/Issue URL: https://app.asana.com/0/1177771139624306/1205069580048345/f
Tech Design URL: 
CC:

**Description**:
This PR fixes receiving of long messages from Bitwarden app

**Steps to test this PR**:
**Test new logic**
1. Make sure you have credentials (can be random) stored for `accounts.google.com`
2. Duplicate credentials (or add new credentials) for `accounts.google.com` so you have at least 20 entries (We need a long message from Bitwarden)
3. Verify autofill of creadentials works on `accounts.google.com`

**Test Bitwarden integration in general**
1. Visit [Bitwarden Integration](https://app.asana.com/0/1202500774821704/1205672027810945) testing template
2. Execute tests and verify all pass



---
###### Internal references:
[Pull Request Review Checklist](https://app.asana.com/0/1202500774821704/1203764234894239/f)
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
[Pull Request Documentation](https://app.asana.com/0/1202500774821704/1204012835277482/f)
